### PR TITLE
components: Pin linux-bridge version

### PR DESCRIPTION
**Pull request by ormergi**
**What this PR does / why we need it**:
Follow-up to https://github.com/kubevirt/cluster-network-addons-operator/pull/1767

Use plugins latest release v1.44.1 [1] for linux-bridge component. 

[1] https://github.com/containernetworking/plugins/releases/tag/v1.4.1

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: data/linux-bridge/002-linux-bridge.yaml [hunk 1]
```diff
@@ -73,3 +73,29 @@ spec:
...
+allowHostPorts: false
+readOnlyRootFilesystem: false
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:kube-system:passt-binding-cni
+volumes:
+- "*"
```
Comment (oshoval) [inline data/linux-bridge/002-linux-bridge.yaml:101]: missing new line

do we need please to wrap with SCC enabled ? as we have on 001-rbac file

**Comment by RamLavi** [comment]
@ormergi what's the status on this PR?

**Comment by RamLavi** [comment]
@ormergi I'm closing this PR. If you need it again, feel free to reopen :)

**Comment by ormergi** [comment]
Sorry for late response, I still think it worth working on it I will post new PR as soon as I have capacity :slightly_smiling_face:

**Pull request closed without merging**